_settings/mappings/views.json

```diff
@@ -28,38 +37,24 @@
     "comment_status": {
       "type": "string"
     },
-    "custom_fields": {
-      "properties": {
-        "popular_posts": {
-          "type": "string"
-        },
-        "related_links_0_link": {
-          "type": "string"
-        },
-        "related_links_1_link": {
-          "type": "string"
-        },
-        "related_links_2_link": {
-          "type": "string"
-        },
-        "related_links_3_link": {
-          "type": "string"
-        },
-        "related_links_4_link": {
-          "type": "string"
-        },
-        "related_post": {
-          "type": "string"
-        }
-      }
-    },
     "date": {
-      "type": "string"
```

Comment:
Setting this as type `date` some issues when deploying to beta (but nowhere else), and we never figured out why.  While I agree that it should be `date`, it's something we need to figure out.